### PR TITLE
[Bugfix] Fix refreshScheme might be NULL

### LINKED ISSUE #10709: In some case show mv will failed
### Steps to reproduce the behavior (Required)
1. SHOW MATERIALIZED VIEW WHERE NAME = 'mv1';

### Expected behavior (Required)
show defails
### Real behavior (Required)
ERROR 1064 (HY000): call frontend service failed, address=TNetworkAddress(hostname=192.168.100.128, port=9020), reason=Internal error processing listTableStatus, host: unknown
### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
2.4

### PR DESCRIPTION
**Pull request by Astralidea**
## What type of PR is this：
- [x] bugfix
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #10709

## Problem Summary(Required) ：

refreshScheme might be NULL in some corner case. We should not report an error in this case.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java [hunk 1]
```diff
@@ -1832,10 +1832,14 @@ public static String getMaterializedViewDdlStmt(MaterializedView mv) {
         DistributionInfo distributionInfo = mv.getDefaultDistributionInfo();
         sb.append("\n").append(distributionInfo.toSql());
 
-        // refresh schema
+        // refresh scheme
         MaterializedView.MvRefreshScheme refreshScheme = mv.getRefreshScheme();
-        sb.append("\nREFRESH ").append(refreshScheme.getType());
-        if (refreshScheme.getType() == MaterializedView.RefreshType.ASYNC) {
+        if (refreshScheme == null) {
```
Comment (ABingHuang) [inline fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java:1837]: I think refreshScheme should not be null.
maybe some bugs lead to this result.
better to set mv to not active and REFRESH type to UNKNOWN

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 3 / 4 (75.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|3|4|75.00%|[1839]|

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 3 / 4 (75.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|3|4|75.00%|[1839]|

**Review by murphyatwork — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Comment by Astralidea** [comment]
@Mergifyio backport branch-2.4
